data.rs

```diff
@@ -52,6 +67,7 @@ impl PSP22Data {
         self.allowances.get((owner, spender)).unwrap_or_default()
     }
 
+    // Transfer `value` tokens from `caller` to `to`.
     pub fn transfer(
```

Comment:
You need tripe slashes (`///`) as double ones won't render as comments on crates.